build.gradle

```diff
@@ -369,7 +369,7 @@ configurations.all {
 }
 
 dependencies {
-  implementation 'com.github.hmcts:civil-commons:1.0.12'
+  implementation 'com.github.hmcts:civil-commons:1.0.12-CIV-7733-v3'
```

Comment:
Just a comment that this will need to be updated before merging.

Reply:
updated to appropriate release version